app/web/utils/timeAgo.ts

```diff
@@ -19,6 +21,9 @@ export interface FuzzySpec {
   text: string;
 }
 
+/**
+ * @deprecated Use `timeAgoI18n` instead.
```

Comment:
Had to do this since `profile` feature area still uses this function and I don't fancy dealing with this for now as part of this...